python/cudf/cudf/core/column/column.py

```diff
@@ -877,6 +878,9 @@ def can_cast_safely(self, to_dtype: Dtype) -> bool:
         raise NotImplementedError()
 
     def astype(self, dtype: Dtype, **kwargs) -> ColumnBase:
+        if dtype in cudf_dtypes_to_pandas_dtypes.values():
```

Comment:
```suggestion
        if dtype in pandas_dtypes_to_cudf_dtypes:
```

Could use an inverse map that already exists.

Reply:
lol, I did not even see that 👍 🤦